src/modules/sandbox/components/SideNav.tsx

```diff
@@ -1,66 +1,72 @@
 import React from 'react';
 import { Box, Grid, Stack, StxInline, StackProps, color } from '@stacks/ui';
 import { border } from '@common/utils';
-import { ClarityIcon } from '@sandbox/components/clarity-icon';
-import { Tooltip } from '@components/tooltip';
+import { ClarityIcon } from '@modules/sandbox/components/ClarityIcon';
 import FunctionIcon from 'mdi-react/FunctionIcon';
 import { DropIcon } from '@components/icons/drop';
-import { Routes } from '@sandbox/common/types';
 import Link from 'next/link';
-import { useConnect } from '@sandbox/hooks/use-connect';
 import { useAppSelector } from '@common/state/hooks';
 import { selectActiveNetwork } from '@common/state/network-slice';
 import { buildUrl } from '@components/links';
+import { useRouter } from 'next/router';
+import { useUser } from '@modules/sandbox/hooks/useUser';
 
 export const SideNav: React.FC<StackProps> = props => {
-  const { isSignedIn } = useConnect();
   const networkMode = useAppSelector(selectActiveNetwork).mode;
+  const router = useRouter();
+  const { isConnected } = useUser();
 
   const navigation: {
     label: string;
     url: string;
     icon: any;
     isDisabled?: boolean;
+    isSelected: boolean;
   }[] = [
     {
       label: 'Write & Deploy Contracts',
       url: buildUrl(`/sandbox/deploy`, networkMode),
       icon: <ClarityIcon size="24px" />,
+      isSelected: router.pathname === '/sandbox/deploy',
     },
     {
       label: 'Call Functions',
       url: buildUrl(`/sandbox/contract-call`, networkMode),
       icon: <Box as={FunctionIcon} size="24px" />,
+      isSelected: router.pathname === '/sandbox/contract-call',
```

Comment:
It looks like the selected state isn't working in the UI for the `contract-call` page, but it is for the other two pages

Reply:
This was just before I had connected my wallet, but it works after I connect the wallet. So not a big deal, but a bit strange

Reply:
<img width="916" alt="Screen Shot 2022-11-08 at 3 57 08 PM" src="https://user-images.githubusercontent.com/2423414/200694000-6b280356-07be-4056-9d0a-8d4d6ebf758c.png">

Reply:
nice catch! it needs to be `/sandbox/contract-call/[[...params]]`